scanpipe/tests/__init__.py

```diff
@@ -284,3 +286,10 @@ def make_dependency(project, **extra):
         "license_key": "mpl-2.0",
     },
 }
+
+scorecard_data = None
+
+data = Path(__file__).parent / "data"
+
+with open(f"{data}/scorecode/scorecard_response.json") as file:
+    scorecard_data = json.load(file)
```

Comment:
This should not be loaded in the module init but as needed in the test function context.